### DISCUSSION TITLE
Caricamento voti nei giorni dove non è presente firma

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#258] [Idee e proposte]
Caricamento voti nei giorni dove non è presente firma

**Comment by trinko** [comment]
Lo ritengo un errore formale: il voto deve essere sempre associato ad una lezione svolta. 
Penso che ci si esponga ad eventuali ricorsi, perché si sa che si appigliano a qualsiasi cosa, e assegnare un voto in un giorno in cui non si è fatto lezione non mi sembra cosa da poco.
